docs/products/platform/billing/_index.md

```diff
@@ -47,6 +47,14 @@ You create a 4GB Compute Instance on the second day of the month, half-way throu
 
 You created a 4GB Compute Instance mid-way through the month and deleted it exactly 5 days later. In total, it was used for 120 hours. Calculating the service fees at the hourly rate, the total is $3.60. Since this is less than the monthly cap, you are indeed billed at the hourly rate and charged $3.60 for your usage.
 
+#### A Service is Resized During the Billing Cycle
+
+Resizing a service, such as a Compute Instance, effectively creates a new billable service. Each of these billable services will appear as separate line items on your monthly invoice. For instance, you create a 4GB Compute Instance prior to the start of the month and resize it to an 8GB Compute Instance mid-way through the month. Your invoice will have each of these services as separate line items. Each line item will reflect the hourly rate for the time the service was active (up to the monthly cap).
```

Comment:
To mirror the note, I think this line:
> Your invoice will have each of these services as separate line items.

Can be more exacting:

> Your invoice will have two services as separate line items for the two different Compute Instance sizes that existed on your account during the billing cycle.

Reply:
Just integrated your changes into this. Thanks!